### DISCUSSION TITLE
Gonna test if health bar and time bar skins will work

### DISCUSSION THREAD

**Discussion by ThatOneSillyGuyDon** [DEMOLITIONDON96/Demolition-Engine#85] [Announcements]
![5B0953A7-9559-45A5-B01E-E7D1DF2FBC6E](https://user-images.githubusercontent.com/87618246/166265388-7ce9ce8e-1df7-4677-bf54-8600d83317d7.jpeg)

**Comment by Theoyeah** [comment]
Oh my god i gonna die
